### DISCUSSION TITLE
Homebrew Curl (60)  SSL certificate problem: unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by dadams1999** [Homebrew/discussions#3108] [Getting started]
Im new to all of this but i keep trying to install homebrew to my work laptop and keep getting this error

 % /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.htm

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.

I have researched and tried some solutions but i either do not understand them or they dont work.
I am on macOS Monterey Version 12.3

**Comment by SMillerDev** [comment]
Probably your work laptop is blocked from accessing Github, you'll need to figure out a way around that for yourself.

**Reply by dadams1999** [reply]
I grabbed the laptop out of the box and configured it as a personal computer.  Is there anything else that comes to mind for the issue?

**Reply by mistydemeo** [reply]
What do you get if you run `curl -vvv -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install`? The `-vvv` should give us some extra error information.

**Reply by dadams1999** [reply]
*   Trying 146.112.49.46:443...
* Connected to raw.githubusercontent.com (146.112.49.46) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (OUT), TLS handshake, Client hello (1):
* (304) (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (OUT), TLS alert, unknown CA (560):
* SSL certificate problem: unable to get local issuer certificate
* Closing connection 0
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (OUT), TLS alert, unknown CA (560):
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
@mistydemeo

**Reply by Bo98** [reply]
Looks like a DNS configuration issue to me. Unless GitHub recently acquired that IP block, that IP block appears to belong to OpenDNS.

**Reply by mistydemeo** [reply]
Looks like it to me too. Can you look into your DNS settings or your firewall/proxy configuration?

**Reply by deepak8910** [reply]
disconnect vpn, it worked for me.

**Comment by deepak8910** [comment]
i was also getting this error on my work laptop, and i disconnected my vpn, and it ran smoothly without any error.
